### ISSUE TITLE
Chrome Driver Scale Parameter only accepts Integers

### ISSUE THREAD

**Issue by djmurf** [jsreport/jsreport-chrome-pdf#1]
Was curious if there was a reason in lib/chrome.js that the scale is being parsed as an Int: 

```
function numberOrUndefined (param) {
  try {
    return parseInt(param)
  } catch (e) {
    return undefined
  }
}
```

When setting the scale to 0.8, it rounds to a 1. 
Could this use parseFloat() instead so the scale can be a float? 

Thanks!

**Comment by pofider** [comment]
Ah, you are right. Thank you. I will fix that.

**Comment by djmurf** [comment]
Thank you sir!

**Comment by pofider** [comment]
Released in 0.2.1
